MMR.Randomizer/Resources/REQ_CASUAL.txt

```diff
@@ -43117,11 +43101,34 @@
       "Id": "CollectibleStrayFairyGreatBay2",
       "RequiredItems": [
         "AreaGreatBayTempleAccess",
-        "GBT Reverse Water Direction",
```

Comment:
I'm not understanding how you can get this fairy without reversing the water direction.

Reply:
All the implementations of GBT Cutscene Diving inherently include a method of reversing the water direction. It was written this way to separate the Zora methods with the special cutscene diving methods.